src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -1,27 +1,39 @@
 import * as React from "react";
-import { ThemeColorOption } from "../../../types";
+import { ThemeColorOption, TitleColorOption } from "../../../types";
 
 import * as s from "./StatsBlock.module.css";
 
 type StatCardProps = {
   number: string;
+  secondaryNumber?: number;
   statement: string;
 };
 
-const StatCard = ({ number, statement }: StatCardProps) => (
+const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   <div className={s.statCard}>
     <div className={s.number}>{number}</div>
+    {secondaryNumber && (
+      <div className={s.secondaryNumber}>{secondaryNumber}</div>
+    )}
     <div className={s.statement}>{statement}</div>
   </div>
 );
 
 type StatsBlockProps = {
   title: string;
+  subtitle?: string;
   statCards: StatCardProps[];
   theme: ThemeColorOption;
+  titleColor?: TitleColorOption;
 };
 
-const StatsBlock = ({ title, statCards, theme }: StatsBlockProps) => (
+const StatsBlock = ({
+  title,
+  subtitle,
+  statCards,
+  theme,
+  titleColor,
```

Comment:
If you add `black` to the title color options, you could even set that to be the default value here so that you don't have to check whether `titleColor` has been set later, as it will always have some value.

Reply:
Do we plan to implement a class selector for `black`?

Reply:
Please disregard my last question.